core/src/main/java/se2/ticktackbumm/core/network/messages/client/CardOpened.java

```diff
@@ -0,0 +1,6 @@
+package se2.ticktackbumm.core.network.messages.client;
+
+public class CardOpened {
```

Comment:
Der Nachricht und den Handlern der Nachricht fehlt die Information, was auf der Karte steht die aufgedeckt wurde. Entweder hier hinzufügen (und in den Handlern) oder `GameUpdate` anpassen und dafür verwenden.